CocoIDE/cocol.py

```diff
@@ -50,81 +51,95 @@
 
 
 class CocoLink(tk.Tk):
-    def __init__(self, master=None, name="cocol", exitroot=False, sym=True):
-        self.master = master
-        self.mainWin = tk.Toplevel(master=master)
+    def __init__(self, master=None, name="cocol", exitroot=False, sym=True) -> None:
+        super().__init__()
+        self.master: tk.Misc = master
+        self.mainWin: tk.Toplevel = tk.Toplevel(master=master)
```

Comment:
not sure why your explicitly hinting all these?

Reply:
I just like to type hint all the variables in __init__. It's probably unnecessary tho.

Reply:
Yeah, please only do ones mypy can't figure out